CommonLib/Error/GREYError.m

```diff
@@ -108,10 +109,7 @@ + (instancetype)errorWithDomain:(NSString *)domain
 }
 
 - (NSString *)description {
```

Comment:
A user can also check for error.localizedDescription. If you NSLog this, it will still print the contents of the errorInfo dictionary. You should also override localizedDescription and print the new GREYError description you are printing, else call `[super localizedDescription];`.

Reply:
Shouldn't this go into a separate PR?

Reply:
Add a TODO for it. This shouldn't impact calling localizedDescription as of now that users are doing.

Reply:
Resolved